assume/units/storage.py

```diff
@@ -97,14 +97,10 @@ def __init__(
             **kwargs,
         )
 
-        self.max_power_charge = (
-            max_power_charge if max_power_charge <= 0 else -max_power_charge
-        )
-        self.min_power_charge = (
-            min_power_charge if min_power_charge <= 0 else -min_power_charge
-        )
-        self.max_power_discharge = max_power_discharge
-        self.min_power_discharge = min_power_discharge
+        self.max_power_charge = -abs(max_power_charge)
+        self.min_power_charge = -abs(min_power_charge)
+        self.max_power_discharge = abs(max_power_discharge)
```

Comment:
in base.py you say these values are positive / negative, why here now abs?

Reply:
This enforces this hardly (due to the minus in front of the abs), to provide the proper behavior even in case of misconfigurations.

Reply:
True!